site/content/docs/cli/npm-packages.md

```diff
@@ -1,17 +1,20 @@
 ---
-title: Using NPM packages
+title: Using NPM packages and local dependencies
 weight: 4
 menu:
   docs:
     parent: "CLI"
 ---
 
 Checkly lets you use JavaScript / TypeScript in your Browser checks and in API check setup & teardown scripts.
-This JavaScript code executes in a runtime environment managed by Checkly.
-This environment has access to specific Node.js versions and NPM packages that are defined in [our runtimes](https://www.checklyhq.com/docs/runtimes/specs/).
+Checks are able to use NPM packages that are defined in [our runtimes](https://www.checklyhq.com/docs/runtimes/specs/), as well as import local JavaScript and TypeScript files.
+
+
+## NPM packages
 
 > Not all NPM packages from NPM are available inside the context of a Check.
 
+The JavaScript code for checks executes in a runtime environment managed by Checkly.
 A runtime consists of a `runtimeId` which you can set at `Project` level or individual `Check` level.
```

Comment:
@clample @tnolet via the UI I believe Runtimes can be set at Group level too. Is there a discrepancy with the CLI or is this incorrect?

Reply:
@ragog you're right, this can be set at the group level as well.

https://github.com/checkly/checkly-cli/blob/c4c97b2417e630106c878913cec5c4985a93bc33/packages/cli/src/constructs/check-group.ts#L50-L53

I'll update this to:

> Runtime versions can be selected by setting a `runtimeId`. This can be configured using the `Check` and `CheckGroup` constructs, and a default value for the project can be set in the [CLI global configuration](https://www.checklyhq.com/docs/cli/project-structure/#global-configuration).